component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/dto/GenerationResponseDTO.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.extension.identity.emailotp.common.dto;
+
+/**
+ * This class represents a model of to generate EMAIL OTP request.
```

Comment:
Not request, response.

Reply:
Fixed in 5fe267f8e9157aeb87e20a328965679356db8a84